tests/unit/test_feed_optimization.py

```
@@ -0,0 +1,210 @@
+    # New item
+    entries.append(MockFeedEntry(
+        "New Item", "http://opt.feed/new", "new_guid"))
+
+    parsed_feed = MockParsedFeed("OptFeed", entries)
+
+    # Capture SQL queries
+    queries = []
+
+    @event.listens_for(Engine, "before_cursor_execute")
+    def before_cursor_execute(
+        conn, cursor, statement, parameters, context, executemany
```

Ruff (0.15.1)

[warning] Unused function argument: `conn`

(ARG001)